packages/language-server/src/test/completion.test.ts

```diff
@@ -749,6 +765,54 @@ suite('Completions', () => {
           },
         )
       })
+      test('@relation(onDelete: |)', () => {
+        assertCompletion(
+          relationDirectiveUri,
+          { line: 66, character: 36 },
+          {
+            isIncomplete: false,
+            items: [
+              { label: 'Cascade', kind: CompletionItemKind.Enum },
+              { label: 'Restrict', kind: CompletionItemKind.Enum },
+              { label: 'NoAction', kind: CompletionItemKind.Enum },
+              { label: 'SetNull', kind: CompletionItemKind.Enum },
+              { label: 'SetDefault', kind: CompletionItemKind.Enum },
+            ],
+          },
+        )
+      })
```

Comment:
Unrelated to PR:
These tests should really be "On schema '... long string | ...`. with cursor at position as marked in string by `|`, autocomplete should suggest the following things for auto completion" - and everything else should be automated by the test system we have.

Reply:
Issue now at https://github.com/prisma/language-tools/issues/849